From 94f01f2730726cc69b107f131fb84aa1dceb32d1 Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Mon, 18 Apr 2011 15:39:19 +0800
Subject: [PATCH 0112/2463] ENGR00142266-1 eSDHC:Add the GPIO CD/WP on MX5x
 boards

sdhc: add always present member into esdhc platform data struct

Signed-off-by: Richard Zhu <r65037@freescale.com>
---
 arch/arm/plat-mxc/include/mach/esdhc.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/esdhc.h b/arch/arm/plat-mxc/include/mach/esdhc.h
index 86003f4..f6f5ca2 100644
--- a/arch/arm/plat-mxc/include/mach/esdhc.h
+++ b/arch/arm/plat-mxc/include/mach/esdhc.h
@@ -22,5 +22,6 @@
 struct esdhc_platform_data {
 	unsigned int wp_gpio;
 	unsigned int cd_gpio;
+	unsigned int always_present;
 };
 #endif /* __ASM_ARCH_IMX_ESDHC_H */
-- 
1.7.7.4

